File: app/src/main/java/nz/co/zsd/floatingvolume/FloatingOverlayApp.java
package nz.co.zsd.floatingvolume;

import com.google.android.material.color.DynamicColors;

public class FloatingOverlayApp extends android.app.Application {
    @Override
    public void onCreate() {
        super.onCreate();
        DynamicColors.applyToActivitiesIfAvailable(this);
    }
}

File: app/src/main/java/nz/co/zsd/floatingoverlay/FloatingOverlayApp.kt
package nz.co.zsd.floatingoverlay

import android.app.Application
import com.google.android.material.color.DynamicColors

class FloatingOverlayApp : Application() {
    override fun onCreate() {
        super.onCreate()
        DynamicColors.applyToActivitiesIfAvailable(this)
    }
}